templates/cisco_ios_show_object-group.textfsm

```diff
@@ -0,0 +1,31 @@
+Value Required,Filldown OGR_TYPE (Service|Network)
+Value Required,Filldown OGR_NAME (\S+)
+Value DESCRIPTION (.+)
+Value NESTED_OGR (\S+)
```

Comment:
Can there be more than one NESTED_OGR? And maybe better served as we list?

Reply:
I'd probably just spell this out as well: `NESTED_GROUPS`

Reply:
```suggestion
Value NESTED_GROUPS (\S+)
```
OK, I agree